lib/accessory.js

```diff
@@ -54,15 +64,19 @@ class Accessory {
   }
 
   setOutlet (uuid, powered, done) {
-    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {
+	  var that = this;
+    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {	
+		that.outlets.map(function(outlet){
```

Comment:
I'm not familiar with the backward apostrophe convention in JS.  I'm glad to change to it if it works.